tests/Cache/FileSystemCacheItemPoolTest.php

```diff
@@ -0,0 +1,159 @@
+<?php
+/*
+ * Copyright 2024 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Auth\Tests\Cache;
+
+use Google\Auth\Cache\FileSystemCacheItemPool;
+use Google\Auth\Cache\TypedItem;
+use PHPUnit\Framework\TestCase;
+use Psr\Cache\CacheItemInterface;
+
+class FileSystemCacheItemPoolTest extends TestCase
+{
+    private string $defaultCacheDirectory = 'cache/';
```

Comment:
nit: make this `.cache`.
```suggestion
    private string $defaultCacheDirectory = '.cache/';
```

What's the best practices with the trailing `/`? I assume it doesn't make a difference either way...